### PR TITLE
Send Events via grpc

### PR DESCRIPTION
**Pull request by majst01**
- [x] https://github.com/metal-stack/metal-api/pull/275

### CODE REVIEW

File: cmd/wait.go [hunk 1]
```diff
@@ -8,26 +8,40 @@ import (
...
 	defer closer.Close()
 
-	c.Emit(event.ProvisioningEventWaiting, "waiting for allocation")
+	e, eventCloser, err := c.NewEventClient()
+	if err != nil {
+		return err
+	}
+	defer eventCloser.Close()
+
+	s, err := e.Send(context.Background(), &v1.EventServiceSendRequest{
```
Comment (Gerrit91) [inline cmd/wait.go:29]: Maybe context with timeout?

Comment (majst01) [reply cmd/wait.go]: done

File: cmd/wait.go [hunk 2]
```diff
@@ -8,26 +8,43 @@ import (
...
-
 func (c *GrpcClient) WaitForAllocation(machineID string) error {
 	client, closer, err := c.NewWaitClient()
 	if err != nil {
 		return err
 	}
 	defer closer.Close()
 
-	c.Emit(event.ProvisioningEventWaiting, "waiting for allocation")
+	e, eventCloser, err := c.NewEventClient()
```
Comment (Gerrit91) [inline cmd/wait.go:23]: Reuse emit?

**Review by Gerrit91 — APPROVED** [review]

File: cmd/wait.go (hunk 2 above)
Comment (majst01) [reply cmd/wait.go]: done

**Pull request merged**